src/helics/core/coreTypeOperations.cpp

```diff
@@ -95,6 +95,7 @@ static const std::unordered_map<std::string, core_type> coreTypes{{"default", co
                                                                   {"websocket", core_type::WEBSOCKET},
                                                                   {"web", core_type::WEBSOCKET},
                                                                   {"inproc", core_type::INPROC},
+                                                                  {"nng", core_type::NNG},
```

Comment:
What does `NNG` stand for?

Reply:
I believe it stands for [Nanomsg Next Generation](https://github.com/nanomsg/nng), which is an alternative to ZMQ, which at some points in the past we had debated creating a core type for, so the type names existed for it, but we haven't had a requirement that drove it in a while.   When we get to some large scales it may be worth a second look since it is MIT licensed, but we will see.  For the time being it was just convenient for this purpose.    For HELICS 3.0 We probably should just make a dummy one has the type and string conversion but it guaranteed to not exist.